GliaWidgets/ViewModel/EngagementViewModel.swift

```diff
@@ -95,6 +99,20 @@ class EngagementViewModel {
 
     func viewDidAppear() {}
 
+    func viewWillAppear() {
+        if isEngagementEndedByOperator {
+            EngagementViewModel.alertPresenters.insert(self)
+            engagementAction?(
+                .showSingleActionAlert(
+                    alertConfiguration.operatorEndedEngagement,
+                    actionTapped: { [weak self] in
+                            self?.endSession()
```

Comment:
probably I could be wrong but looks like an incorrect alignment of `self?.endSession()`.